### DISCUSSION TITLE
GERS and wikidata

### DISCUSSION THREAD

**Discussion by wipfli** [OvertureMaps/data#315] [Discussion]
I think it would be nice if GERS ids could be mapped to wikidata items. This would open up a lot of practical opportunities like for example usage of QRank (popularity measure of a wikidata item).

Do you have any plans/thoughts on GERS and wikidata conflation?

**Comment by varapmsft** [comment]
Thanks @wipfli for the suggestion. While we do not have explicit plans for mapping GERS ids to wikidata items at this point, there are two ways to generate these mappings as mentioned in the [documentation](https://docs.overturemaps.org/gers/#obtaining-a-gers-id).

- In option 1, WikiData could become a source feed for Overture assuming licensing requirements are reviewed and we are allowed to use the data _and_ the data meets the quality bar. Regarding the quality bar, we are still working on measurement process for evaluating the quality of prospective input feeds.
- In option 2, when stable GERS ids are available in Overture data sets, anyone in the community can run conflation of Overture data with wikidata and produce these mappings. We do have future plans to share example code snippets/scripts to help community with conflation.

**Reply by Cj-Malone** [reply]
> We do have future plans to share example code snippets/scripts to help community with conflation.

That's great news. Thank you :)

> Wikidata

Wikidata has notability requirements, so the majority of Overture features wont have a match. However it's massively beneficial link the to datasets in specific ways, like you are with places brands (Presumably via NSI?).
